Chris' branch

reevesglobal opened the pull request:
Started the HowItWorks Component (used the Styled SectionTitle) and made a reusable InfoCard Component - which I found and edited from this tutorial here: https://jsmanifest.com/build-reusable-responsive-react-card-with-styled-components/

Confidenceiskey commented:
Awesome, thanks @reevesglobal! Can you get rid of the package.lock file that seems to have nothing in there but 3 lines. And can you also make the SectionTitle padding-top dynamic too? Like how it is with the color?

reevesglobal commented:
Ok requests completed :D I also changed my HowItWorks component to be responsive - deleted hard-coded "height" and added "flex-flow: wrap;". Let me know if I should make more properties of the InfoCard dynamic.

Confidenceiskey commented:
I will close this and use your other PR
